cookbooks/bcpc-hadoop/recipes/journalnode.rb

```diff
@@ -32,6 +32,24 @@
     action :create
     recursive true
   end
+
+ if get_config("jn_txn_fmt") then
```

Comment:
This block should be done outside the for loop (though due to the `not_if` guard it will be done only once regardless of the duplicate attempts to create file resources).